### DISCUSSION TITLE
Is there a way to disable github issues on one or all sites

### DISCUSSION THREAD

**Discussion by KlutzyBubbles** [upptime/upptime#810] [Q&A]
I have been looking through the documentation, issues and discussions and cant find the question i am looking for (please link if it exists andi just havent found it).

I am looking to disable issues created for one or all of the sites i am monitoring, as i am using this to check sites that are on at a glance and they are definitely not on or working most of the time so i just get flooded with created and closed issues.

**Comment by AnandChowdhary** [comment]
Unfortunately this is currently unsupported :(
